src/components/ArticleById/ArticleById.jsx

```diff
@@ -1,10 +1,27 @@
 import {useParams} from 'react-router-dom'
 import { useState, useEffect } from 'react';
-import { getArticleById } from '../../api';
+import { getArticleById, patchVotesByArticleId } from '../../api';
 import './ArticleById.css'
 import CommentsById from '../CommentsById/CommentsById';
 
 function ArticleById(){
+
+    function handleVote(e, numOfVotes){
+        const voteObj = { in_votes: numOfVotes };
```

Comment:
should this not be inc_votes? I am getting 400 bad request errors when trying to vote I think this might be causing it.